test/test_core.rb

```diff
@@ -1323,11 +1324,26 @@ def test_ticket_21
       exception = assert_raise(ParseException) do
         Document.new(src)
       end
-      assert_equal(<<-DETAIL, exception.to_s)
+      assert_equal(<<-DETAIL.chomp, exception.to_s)
 Missing attribute value start quote: <bar>
 Line: 1
 Position: 16
 Last 80 unconsumed characters:
+value/>
+      DETAIL
+    end
+
+    def test_ticket_21_part_2
```

Comment:
Could you use meaningful test name?

The `ticket_21` refers the ticket ID of REXML's original bug trucker, which can't be referred now. :<

Reply:
OK.
I see.

I would change it to `test_parse_exception_on_missing_attribute_end_quote`.